bg_atlasgen/atlas_scripts/azba_zfish.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Script to generate a Brainglobe compatible atlas object for the Adult Zebrafish Brain Atlas (AZBA)
+
+@author: Kailyn Fields, kailyn.fields@wayne.edu
+
+"""
+
+__version__ = "0"
+
+import csv
+import time
+import tifffile
+import tarfile
+
+import numpy as np
+import multiprocessing as mp
+
+from rich.progress import track
+from pathlib import Path
+from bg_atlasgen.mesh_utils import Region, create_region_mesh
+from bg_atlasapi.structure_tree_util import get_structures_tree
+from bg_atlasgen.wrapup import wrapup_atlas_from_data
+from bg_atlasapi import utils
+
+PARALLEL = False #Disable for debugging mesh creation
+    
+def create_atlas(working_dir, resolution):
+    
+    #metadata
+    ATLAS_NAME = "azba_zfish"
+    SPECIES = "Danio rerio"
+    ATLAS_LINK = "http://www.azba.wayne.edu"
+    CITATION = "Kenney et al. 2021, https://doi.org/10.1101/2021.05.04.442625"
+    ATLAS_FILE_URL = "http://www.azba.wayne.edu/2021-08-22_AZBA.tar.gz"
+    ORIENTATION = "las"
+    ROOT_ID = 9999 
+    ATLAS_PACKAGER = "Kailyn Fields, kailyn.fields@wayne.edu"
+    ADDITIONAL_METADATA = {}
+    
+    
+    #setup folder for downloading
```

Comment:
To ensure that >1 atlas can be created at the same time I'd add this line here:
```python
working_dir = working_dir / "brainglobe_workingdir" / ATLAS_NAME
```

Reply:
Even though working_dir is already working_dir / "brainglobe_workingdir" ?

Reply:
Ah good catch, sorry it should be:
```python
working_dir = working_dir  / ATLAS_NAME
```